cmake/Platform/Common/GCC/Configurations_gcc.cmake

```diff
@@ -72,7 +72,6 @@ ly_append_configurations_options(
         -Wno-stringop-truncation
         -Wno-switch
         -Wno-uninitialized
-        -Wno-unused-but-set-variable
```

Comment:
question: This warning has helped to improve the code by removing unused variables (which could be a bug), are we sure we want to remove it? Maybe we can add it to the other compilers so AR picks them up too? thoughts?

Reply:
Silly me, you're right. I misread it. Cheers.